modules/cms/models/maintenancesetting/fields.yaml

```diff
@@ -15,4 +15,21 @@ fields:
 
     cms_page:
         type: dropdown
-        cssClass: checkbox-align
\ No newline at end of file
+        cssClass: checkbox-align
+
+    ip_whitelist:
+        label: cms::lang.maintenance.ip_whitelist.name
+        type: repeater
+        cssClass: checkbox-align
+        form:
+            fields:
+
+                ip:
+                    label: cms::lang.maintenance.ip_whitelist.ip
+                    type: text
+                    span: left
+
+                label:
+                    label: cms::lang.maintenance.ip_whitelist.ip
```

Comment:
`cms::lang.maintenance.ip_whitelist.label` ?

Reply:
Nice spot, totally missed that :+1: